snapshots/artificial_intelligence/2023-06-14/annual_count_industrial_robots_2011_2021.csv.dvc

```diff
@@ -0,0 +1,5 @@
+wdir: ../../../data/snapshots/artificial_intelligence/2023-06-14
```

Comment:
Are these files with only `wdir` and `outs` meant to be here?

Reply:
this happens because I am running through multiple urls in snapshot but actually I won't have any of these individual files as I am concatenating them all into one called ai_robots.csv - not sure how I get rid off these? or if they are intended to be somewhere else?

Reply:
will try to remove them too :)